### PR TITLE
feat(scripts): add WithdrawTokens for bot wallet consolidation

### PR DESCRIPTION
**Pull request by pemulis**
Signed-off-by: John Shutt <john.d.shutt@gmail.com>

**Motivation**

Plan to use this script to transfer all of the tokens from our current multi-collateral bot to be redeemed by sponsor wallets, as we're consolidating to a single-collateral bot system.

**Summary**

This is a stripped-down version of the `WithdrawTokensFromDSProxy` script in the `trader` package.

**Details**

n/a

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [x]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

n/a

### CODE REVIEW

**Comment by pemulis** [comment]
Closing for a moment to run some mainnet fork tests. Will re-open when ready for review.

**Comment by pemulis** [comment]
Looks good on mainnet fork with `accounts[0]` replaced with the hardcoded wallet address.
image

**Review by chrismaree — APPROVED** [review]
LGTM! Nice simple script

File: packages/core/scripts/mainnet/WithdrawTokens.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  // Figure out how many tokens to withdraw. If max, then query the full balance of the unlocked account. Else, use specified.
+  const amountToWithdraw = (argv.amount == "max" ? await token.balanceOf(accounts[0]) : argv.amount).toString();
+
+  // Send the transaction against the DSProxy manager.
+  const tx = await token.transfer(argv.recipientAddress, amountToWithdraw, {
+    from: accounts[0],
+    gasPrice: gasEstimator.getCurrentFastPrice()
+  });
+
+  console.log(`Sent ${amountToWithdraw} ${await token.symbol()} to ${argv.recipientAddress}.`);
```
Comment (chrismaree) [inline packages/core/scripts/mainnet/WithdrawTokens.ts:56]: probs want to cast `amountToWithdraw` to a string to make it more readable.

**Pull request merged**